FRENCH/src/SUMMARY.md

```diff
@@ -13,3 +13,5 @@
     - [Hello, Cargo!](ch01-03-hello-cargo.md)
 
 - [Programmer un jeu de devinettes](ch02-00-guessing-game-tutorial.md)
+
+- [Une structure de contrôle concise avec `if let`](ch06-03-if-let.md)
```

Comment:
`avec` should be used when referring to the concept, not the structure itself
```suggestion
- [Une structure de contrôle concise: `if let`](ch06-03-if-let.md)
```
Or
```suggestion
- [Contrôle de flot concis avec `if let`](ch06-03-if-let.md)
```